src/components/ProfileDisplay.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import CommunityGrid from './CommunityGrid';
+import StudentGrid from './StudentGrid';
+
+export default function ProfileDisplay() {
+  const [user, setUser] = React.useState(0);
+
+  const handleChange = (event, newUser) => {
+    setUser(newUser);
+  };
+
+  function displaySelection(userSelected) {
+    if (userSelected === 0) {
+      return <StudentGrid />
+    }
+    return <CommunityGrid />
+  }
+
+
+  return (
+    <div className="stepperStyle">
+      <Paper square>
+        <Tabs value={user} indicatorColor="primary" textColor="primary" onChange={handleChange}>
```

Comment:
There's a missing [Typography](https://material-ui.com/components/typography/) element above this that says **Scholarships Made Simple**

Reply:
Ok good catch, it is on there now.